matching-service/utils/socket-io.js

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Curious what's the rationale behind prohibiting this?

Reply:
I am unable to assign `socket.userId = payload.username;` unless I disable the rule. Disabled it for the file in case we plan to add other information to the socket, I feel that reassignment should be allowed in this case.

Rule:
`(parameter) socket: Socket<DefaultEventsMap, DefaultEventsMap, DefaultEventsMap, any>
Assignment to property of function parameter 'socket'.eslint([no-param-reassign](https://eslint.org/docs/rules/no-param-reassign))`